oct/pages/models/add_address_page.py

```diff
@@ -0,0 +1,53 @@
+from pages.base.page import BasePage
+from pages.base.elements import RadioButton, Form, InputField, RadioButtonGroup, DropDown
+
+
+class DefaultAddress(RadioButtonGroup):
+    contains = {
+        "yes": {
+            "locator": ("XPATH", "div/label[1]/input"),
```

Comment:
Relative xpath starts with "//" (the same issue in line 12). And div seems to be redundant in both lines:
```suggestion
            "locator": ("XPATH", "//label[1]/input"),
```

Reply:
Unfortunately, github do not have "handface" smile :D

Reply:
Unfortunately, github do not have "handface" smile :D

It should be only one "/" in the path, no?: "/label[1]/input"

Reply:
Probably "/" should work too. But xpath shouldn't start without "/" or "//" slashes.

> Single Slash “/” – Single slash is used to create Xpath with absolute path i.e. the xpath would be created to start selection from the document node/start node.
> Double Slash “//” – Double slash is used to create Xpath with relative path i.e. the xpath would be created to start selection from anywhere within the document.

Reply:
done